Makefile

```diff
@@ -16,21 +16,40 @@ help:
 	@echo "docker-test-build - build test container"
 	@echo "docker-test-shell - run shell from a test container"
 
+out/json-schemas.sql: schema/sql/json-schemas.sql
+	mkdir -p $(@D)
+	./scripts/build schema < "$<" > "$@"
+
 .PHONY: build
-build: clean fetch
-	./scripts/build schema/ out/
+build: out/json-schemas.sql
+
+schema/ext/json-schema/%:
+	git submodule update --init --recursive $(@D)
+
+test/sql/plpgunit/%:
+	git submodule update --init --recursive $(@D)
 
 .PHONY: fetch
-fetch:
-	git submodule update --init --recursive
+fetch: schema/ext/json-schema/ test/sql/plpgunit/
 
 .PHONY: fetch-latest
 fetch-latest:
 	git submodule foreach 'git checkout master && git pull'
 
+SCHEMA_FILES := \
```

Comment:
So going forward we're going to need to maintain this list, correct?

Reply:
we already did. previously this list was in `scripts/build` as `sql_names`.

Reply:
Ah yes, https://github.com/hashbang/userdb/pull/52/files#diff-cc1bc65976b3ec6248bb0079f2f78eeab12f1f9da665739637b0cb576f816c06L7-L14